### ISSUE TITLE
RcrtES_noPIN_v1r0 'No PIN' var not setting to 1

### ISSUE THREAD

**Issue by robertsamm** [episphere/connectApp#352]
In UofC stage testing they had a participant with Connect ID 7912819503 that came in passively and did not enter a PIN. Var for 'PIN entered=blank' but the var for 'No PIN=No'. This is incorrect as 'No PIN' should =Yes, if a recruit clicks the "I don't have a pin" should be set to 1. 

No PIN var is RcrtES_noPIN_v1r0

**Comment by cunnaneaq** [comment]
I tested this in dev today and I was able to click 'submit' without entering a PIN in the box.  

Another solution suggested by @FrogGirl1123  in addition to @robertsamm's is to only allow participants to move forward if they click "I don't have a PIN" or enter a PIN and click "submit" but not able to click submit if the PIN box is empty.  I pasted a screenshot of the PIN page here for reference.

![Screen Shot 2022-05-18 at 4 46 04 PM](https://user-images.githubusercontent.com/83971268/169152774-550cc296-c53f-4cbc-b04a-adbba448a842.png)
